app/main.py

```diff
@@ -1,3 +1,86 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from app.shop import Shop
+from app.customer import Customer
+
+
+def shop_trip() -> None:
+    with open("app/config.json", "r") as f:
+        data = json.load(f)
+        for customer in data["customers"]:
+            cust = Customer(
+                name=customer["name"],
+                products=customer["product_cart"],
+                location=customer["location"],
+                money=customer["money"],
+                car=customer["car"],
+            )
+            print(f"{cust.name} has {cust.money} dollars")
+            closest = {}
+            shop_names = {}
+            cust_prod = []
+            prod_price = []
+            sh_prod_names = []
+            prod_count = []
+            sum_all = []
```

Comment:
Why do you need all those empty variables here. I tried to find fast where you use one of them, but I couldn’t. Try to find the way to do not declare empty variables here

Reply:
how can I  get rid of these empty variables, for example? Because of nested loops, I need a place to collect data for further iterations, you know.